src/coffea/nanoevents/factory.py

```diff
@@ -252,7 +245,7 @@ def from_root(
         access_log=None,
         iteritems_options={},
         use_ak_forth=True,
-        permit_dask=False,
+        permit_dask=True,
```

Comment:
This is a big API change, but it is coming in a yet-unreleased major coffea version change, so it is ok. It also helps users to be exposed to dask. My only complaint is that the argument name is still a bit strange to me, rather I would think `delayed=True` or `eager_mode=False` would be better?

Reply:
I think `delayed=True` makes more sense. I'll change this.